### ISSUE TITLE
Implement the image resize handler

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5#5167]
The functionality of the [enhanced image plugin in v4](https://ckeditor.com/cke4/addon/image2) is missing in v5.

![image](https://user-images.githubusercontent.com/156149/47722540-ed367c00-dc52-11e8-97d4-b7eafbb6e495.png)

A follow-up of https://github.com/ckeditor/ckeditor5/issues/1330.

---

If you'd like to see this feature implemented, add 👍 to this post.

**Comment by adrianmihaila** [comment]
Hello, I and my colleague just released a plugin that allows you to edit the size of an image using width and height input, as well as providing an aspect ratio lock tool that makes it easy to scale without distorting the image. 
The plugin is here: https://github.com/eMAGTechLabs/ckeditor5-classic-image-resize. 
You can give it a try :pray::pray:. Also, any suggestions or feedback are welcome :thumbsup::thumbsup:.

**Comment by oleq** [comment]
@adrianmihaila Give a thumbs up in #5198 if you want this to be a core feature 😀

**Comment by vineetsingh065** [comment]
@mlewand  Hi is the image resize handler plugin added into django ckeditor?

**Comment by mlewand** [comment]
@vineetsingh065 We're not maintaining any CKE5 Django integration, so you'd need to ask this the package maintainer.

That being said, even if such integration doesn't support it, CKE5 is a plain JavaScript integration so you can make your own build using our online builder and for sure there's a way to include a js library in Django.

**Comment by joelpittet** [comment]
Drupal is considering changing the default to `px` here BTW https://www.drupal.org/project/drupal/issues/3348603 See the examples for why `%` posses a problem for `figcaption`, image alignment, and cached serverside resize solutions.

**Comment by joelpittet** [comment]
Lots come down to a common practice for responsive images to follow: `img { max-width: 100%; height: auto; }` and the prevelence of bootstrap https://getbootstrap.com/docs/5.3/content/images/#responsive-images

**Comment by Witoso** [comment]
Hey @joelpittet, thanks for reaching out! I already answered on the Drupal issue, cross-posting some information here for the reference. We chose \`%\`, as a lot of people may not know the exact size of the container in which the content will be used. Often, content is used in multiple destinations at once. With pixels, one gets far less flexibility with how the content will be used in the future.

As for the responsive option for the image, let us discuss it internally, and I will get back to you as soon as possible.
